### PR TITLE
adding numconv recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: 
https://github.com/nsls-ii-forge/numconv-feedstock
https://github.com/nsls-ii-forge/sirepo-bluesky-feedstock
https://github.com/nsls-ii-forge/bloptools-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes Hello, I need this merged so two other PR can be completed successfully.

**Review by dopplershift — APPROVED** [review]
Recipe looks fine. Need @mrakitin to confirm that they are willing to be a maintainer for this recipe.

**Comment by dopplershift** [comment]
@kbeeperez Did something go wrong here?

**Comment by kbeeperez** [comment]
@dopplershift no,  I just added sirepo-bluesky & bloptools recipes to package them in one shot per @mrakitin suggestion because they rely on numconv.

**Comment by dopplershift** [comment]
Ah, GitHub was only showing me the most recent 2 before...now I see them all.

**Comment by mrakitin** [comment]
> Recipe looks fine. Need @mrakitin to confirm that they are willing to be a maintainer for this recipe.

Sorry for the delay, was on vacation. Confirming my desire to be a maintainer.

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes should be ready to be merged

File: recipes/bloptools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - bluesky >=1.6.1
+    - databroker
+    - matplotlib
```
Comment (dopplershift) [inline recipes/bloptools/meta.yaml:26]:
```suggestion
    - matplotlib-base
```

File: recipes/sirepo-bluesky/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - bluesky
+    - databroker
+    - ipython
+    - matplotlib
```
Comment (dopplershift) [inline recipes/sirepo-bluesky/meta.yaml:27]:
```suggestion
    - matplotlib-base
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple `matplotlib`->`matplotlib-base` changes and these are ready.

**Review by mrakitin — APPROVED** [review]

**Review by dopplershift — APPROVED** [review]

**Comment by mrakitin** [comment]
Thanks, @dopplershift!

**Pull request merged**